index.js

```diff
@@ -1 +1 @@
-module.exports = require('./lib/Logger.js');
+module.exports = { Logger: require('./lib/Logger.js') };
```

Comment:
Why did you change that, is it not simpler to call the logger the same way as before ?